pkg/forwarder/transaction.go

```diff
@@ -105,22 +129,41 @@ var defaultAttemptHandler = func(transaction *HTTPTransaction) {}
 var defaultCompletionHandler = func(transaction *HTTPTransaction, statusCode int, body []byte, err error) {}
 
 func initTransactionExpvars() {
+	transactionsInputBytes.Init()
+	transactionsConnectionEvents.Init()
+	transactionsInputCount.Init()
+	transactionsDroppedByEndpoint.Init()
+	transactionsRequeuedByEndpoint.Init()
+	transactionsRetriedByEndpoint.Init()
+	transactionsSuccessByEndpoint.Init()
+	transactionsSuccessBytes.Init()
 	transactionsErrorsByType.Init()
 	transactionsHTTPErrorsByCode.Init()
-	transactionsExpvars.Set("RetryQueueSize", &transactionsRetryQueueSize)
-	transactionsExpvars.Set("Success", &transactionsSuccessful)
+	transactionsConnectionEvents.Set("DNSSuccess", &connectionDNSSuccess)
+	transactionsConnectionEvents.Set("ConnectSuccess", &connectionConnectSuccess)
+	transactionsExpvars.Set("InputBytes", &transactionsInputBytes)
+	transactionsExpvars.Set("ConnectionEvents", &transactionsConnectionEvents)
+	transactionsExpvars.Set("InputCount", &transactionsInputCount)
```

Comment:
for consistency with other expvars, you could name this expvar `InputCountByEndpoint`.

Ditto for `InputBytes` and `SuccessBytes`